### DISCUSSION TITLE
Thermostate state: ON

### DISCUSSION THREAD

**Discussion by akloeckner** [SukramJ/aiohomematic#1065] [Ideas]
Hi, now with increasing temperatures, we have switched off the heating in our house. It seems good practice to open all thermostate valves in this case. HM offers an `on` (or: "An") state for this. It seems to set the target temperature to 30 °C or 30.5 °C depending on the model of the thermostate. My thermostates are shown in HA as `heating` now, with this maximum target temperature. 

I wonder, if there is a better way to represent that `on` state in HA. After all, there is an `off` state. So, maye it's worth to also provide an `on` state. That way, this mode could also be triggered from within HA, which I have no idea how to do it right now. A use case would be to switch the house to "summer mode".

**Comment by SukramJ** [comment]
The status ON does not exist in the HA [climate data model](https://developers.home-assistant.io/docs/core/entity/climate#hvac-modes).
If you want a change in the data model of HA you have to start a discussion in the [HA Archtekture repo](https://github.com/home-assistant/architecture/discussions).

**Comment by akloeckner** [comment]
Oh ok... It will be difficult then to show this state in HA. 

I don't think, it's worth to add a discussion about that. The folks at HA have their own agenda and my experience is not very responsive with such discussions. :-/ Also, in fact, the device is `heating` when `on`. So, it might not even be *right* to change the data model.

Is there a way to set the `on` mode from within HA? E.g. using a setpoint above 30°C? Or a special `set_parameter` call?

**Comment by SeriousPat** [comment]
I made a Script at the Automation Section... with only 2 actions...

Change to Summermode...

![image](https://github.com/danielperna84/hahomematic/assets/98743842/17b39e3f-a90a-4b88-a3fc-4c9f8702269d)
![image](https://github.com/danielperna84/hahomematic/assets/98743842/530ed2de-de98-44a2-b438-0f7de59a1821)

Change to Wintermode...
![image](https://github.com/danielperna84/hahomematic/assets/98743842/b1a1406f-c172-4059-83bb-6147eb14eacc)

**Comment by akloeckner** [comment]
Thanks for the hint! I just tried and setting the temperature to 30.5 actually turns the thermostats ON. So, that's exactly the kind of workaround I was hoping for. 👍
